csq.py

```diff
@@ -1,19 +1,33 @@
-#!/usr/bin/env python3.10
+#!/usr/bin/env python3
```

Comment:
You should set python to python3.10 else it will work since match statement was introduced in python3.10

Reply:
I instead checked that the version is either 3.10 or above. I personally don't have python 3.10, I have python 3.11 so using this fails because the specified environment can't be found.

This is shown in [csq.py line 17](https://github.com/CsqLang/Csq/blob/bash/csq.py#L17) to [csq.py line 29](https://github.com/CsqLang/Csq/blob/bash/csq.py#L29) and first ran before handling any command as seen in [csq.py line 262](https://github.com/CsqLang/Csq/blob/bash/csq.py#L262)

Reply:
so we can use python3.11 instead it will work as i have tested....

Reply:
No need to change the environment to include the minor version, let's leave it as 'python3' and the version check will do the rest

Reply:
but if someone is having multiple versions in his env so its a bit problem i am getting it...

Reply:
If a person has multiple versions, the one(s) above or python3.10 will be chosen, whichever one is default. This reduces the need to know each persons environment and offload the work to python to figure out what to do.

Reply:
In my case it's different the Python is chosing version < 3.10

Reply:
Your default python version is < 3.10, that means it will be used, for now the only way is to [change the default version](https://unix.stackexchange.com/questions/410579/change-the-python3-default-version-in-ubuntu) to >= 3.10 then try again before a permanent solution is found.